src/pages/studyView/tabs/SummaryTab.tsx

```diff
@@ -113,11 +113,20 @@ export class StudySummaryTab extends React.Component<
                 chartMeta: ChartMeta,
                 dataBins: GenericAssayDataBin[]
             ) => {
-                this.store.updateGenericAssayDataIntervalFilters(
+                this.store.updateGenericAssayDataFilters(
                     chartMeta.uniqueKey,
                     dataBins
                 );
             },
+            onGeneircAssayValueSelection: (
```

Comment:
typo - Generic

Reply:
oops... Thanks! I just updated it: https://github.com/cBioPortal/cbioportal-frontend/pull/3870/commits/6b9708f6c993eae61b5773e9adf028538fc66e75